Stripe/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -46,34 +55,49 @@
 /* Caption for Email field on address form */
 "Email" = "Email";
 
+/* Source type brand name */
+"EPS" = "EPS";
+
 /* accessibility label for text field */
 "expiration date" = "expiration date";
 
 /* Label for free shipping method */
 "Free" = "Free";
 
+/* Source type brand name */
+"Giropay" = "Giropay";
+
+/* Source type brand name */
+"iDEAL" = "iDEAL";
+
 /* Shipping form error message */
 "Invalid Shipping Address" = "Invalid Shipping Address";
 
 /* Title for screen when data is still loading from the network. */
 "Loading…" = "Loading…";
 
+/* Source type brand name */
+"Multibanco" = "Multibanco";
+
 /* Caption for Name field on address form */
 "Name" = "Name";
 
 /* Button to move to the next text entry field */
 "Next" = "Next";
 
-/* No comment provided by engineer. */
+/* ok button */
 "OK" = "OK";
 
+/* Source type brand name */
+"P24" = "P24";
+
 /* Title for Payment Method screen */
 "Payment Method" = "Payment Method";
 
 /* Caption for Phone field on address form */
 "Phone" = "Phone";
 
-/* Short string for postal code (text used in non-US countries). Please keep to 8 characters or less if possible. */
```

Comment:
Were these character count suggestions removed because you re-ran `genstrings`? I wonder if we need to move them into the `STPLocalizedString` call